paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/custom_flow/CustomFlowViewModel.kt

```diff
@@ -0,0 +1,128 @@
+package com.stripe.android.paymentsheet.example.samples.ui.custom_flow
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.github.kittinunf.fuel.Fuel
+import com.github.kittinunf.fuel.core.extensions.jsonBody
+import com.google.gson.Gson
+import com.stripe.android.paymentsheet.PaymentSheetResult
+import com.stripe.android.paymentsheet.example.samples.model.CartState
+import com.stripe.android.paymentsheet.example.samples.networking.ExampleCheckoutResponse
+import com.stripe.android.paymentsheet.example.samples.networking.toCheckoutRequest
+import com.stripe.android.paymentsheet.model.PaymentOption
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.flow.updateAndGet
+import kotlinx.coroutines.launch
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
+import com.github.kittinunf.result.Result as ApiResult
+
+internal class CustomFlowViewModel : ViewModel() {
+
+    val status = MutableLiveData<String?>()
```

Comment:
```suggestion
    val status = MutableStateFlow<String?>()
```

Reply:
That’s actually not used anymore, so I’ll get rid of it.